megamek/src/megamek/client/ui/swing/boardview/AttackSprite.java

```diff
@@ -402,16 +406,19 @@ public void addWeapon(SearchlightAttackAction attack) {
 
     @Override
     public StringBuffer getTooltip() {
-        StringBuffer tipString = new StringBuffer();
-        tipString.append("<FONT COLOR=#");
-        tipString.append(Integer.toHexString(attackColor.getRGB() & 0xFFFFFF));
-        tipString.append(">");
-        tipString.append(attackerDesc
-                + "<BR>&nbsp;&nbsp;" + Messages.getString("BoardView1.on") + "&nbsp;&nbsp;" + targetDesc);
-        tipString.append("</FONT>");
-        for (String wpD: weaponDescs) {
-            tipString.append("<BR>"+wpD);
+        GamePhase phase = this.boardView1.game.getPhase();
+        String result = "";
+        String f = "";
```

Comment:
Please use more descriptive variable names. This provides no information on what this variable does.